### PR TITLE
fix: change high precision money display

### PR DESCRIPTION
**Pull request by montezume**
#### Summary

Currently, when a price is `highPrecision`, the MoneyField "jumps", as it is rendering a label above the field. This is not desired. Instead, we replace it with the use of an icon, and an accompanying tooltip in the MoneyInput.

#### Before

beforemf

#### After

aftermf

Also adds new `FractionDigitsIcon`

### CODE REVIEW

File: src/components/tooltip/README.md [hunk 1]
```diff
@@ -111,3 +111,4 @@ In the following example, the tooltip text only appears on hover when the button
...
 | `children`             | `node`   |    ✅    | -                                                                                                                                            | -       | Content rendered within the tooltip                                                             |
 | `components`           | `object` |    -     | `WrapperComponent`, `BodyComponent`                                                                                                          | -       | If passed, the tooltip will wrap your component with this element                               |
+| `customBodyStyles`     | `object` |    -     | -                                                                                                                                            | -       | If passed, these styles will be spread onto the div surrounding the tooltip body                |
```
Comment (tdeekens) [inline src/components/tooltip/README.md]: Not sure if I am overly picky with the name again. But given we tend to like the `components={{header, footer}}` pattern. How about going with `styles={ body }` here?

File: src/components/inputs/money-input/money-input.js [hunk 2]
```diff
@@ -593,31 +607,81 @@ class MoneyInput extends React.Component {
...
+                    },
+                  });
+                this.setState({ amountHasFocus: true });
+              }}
+              value={this.props.value.amount}
+              css={[
+                getAmountInputStyles({ ...this.props, hasFocus }),
+                // accounts for size of icon
+                css`
+                  padding-right: 24px;
```
Comment (islam3zzat) [inline src/components/inputs/money-input/money-input.js]:
```suggestion
                  padding-right: ${vars.spacing24};
```

File: src/components/tooltip/README.md [hunk 3]
```diff
@@ -111,3 +111,4 @@ In the following example, the tooltip text only appears on hover when the button
...
 | `children`             | `node`   |    ✅    | -                                                                                                                                            | -       | Content rendered within the tooltip                                                             |
 | `components`           | `object` |    -     | `WrapperComponent`, `BodyComponent`                                                                                                          | -       | If passed, the tooltip will wrap your component with this element                               |
+| `styles.body`          | `object` |    -     | -                                                                                                                                            | -       | If passed, these styles will be spread onto the div surrounding the tooltip body                |
```
Comment (islam3zzat) [inline src/components/tooltip/README.md]: the proptypes says it's required.
Should we update the propTypes, or the readme?

File: src/components/inputs/money-input/money-input.js (hunk 2 above)
Comment (montezume) [reply src/components/inputs/money-input/money-input.js]: 👍

File: src/components/tooltip/README.md (hunk 3 above)
Comment (montezume) [reply src/components/tooltip/README.md]: it's required because it's set with a default though, so for the user it's not required.

Comment (islam3zzat) [reply src/components/tooltip/README.md]: thanks for the explanation

File: src/components/inputs/money-input/money-input.styles.js [hunk 4]
```diff
@@ -29,4 +29,20 @@ const getAmountInputStyles = props => [
   `,
 ];
 
-export { getCurrencyLabelStyles, getAmountInputStyles };
+const getHighPrecisionWrapperStyles = props => css`
```
Comment (tdeekens) [inline src/components/inputs/money-input/money-input.styles.js]: Nit just pass `isDisabled`? to be more specific? Not sure what the pattern is on UIKit.

Comment (montezume) [reply src/components/inputs/money-input/money-input.styles.js]: 👍

**Pull request merged**